Dashboard/Utils/GaugeConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Avalonia.Data.Converters;
+using System;
+
+namespace Dashboard.Utils;
+
+public class GaugeConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
```

Comment:
can u fix these warnings? should just be:
```suggestion
    public object Convert(object? value, Type targetType, object? parameter, System.Globalization.CultureInfo culture)
```

Reply:
https://github.com/nickowen44/AV-Datalogger/pull/14/commits/f7226b894ba1d9892b0e536c0077ef14c7bd247b